tests/Google/Ads/GoogleAds/Lib/V3/GoogleAdsClientBuilderTest.php

```diff
@@ -80,6 +82,15 @@ public function testBuildClientFromConfiguration()
         $this->assertSame(self::$LOGIN_CUSTOMER_ID, $googleAdsClient->getLoginCustomerId());
         $this->assertSame('https://abc.xyz:443', $googleAdsClient->getEndpoint());
         $this->assertSame('https://localhost:8080', $googleAdsClient->getProxy());
+        $this->assertSame(self::provideValidTransports()[0][0], $googleAdsClient->getTransport());
+    }
+
+    public function provideValidTransports()
```

Comment:
You can move this down to close to `provideInvalidProxyURIs()`. That's why we use this.